grakn-engine/src/main/java/ai/grakn/engine/Grakn.java

```diff
@@ -40,6 +42,9 @@
      */
     public static void main(String[] args) {
         try {
+            GraknPid pidFile = newPidFile_deleteOnExit();
+            pidFile.createPidFile_deleteOnExit();
```

Comment:
If we move these lines into `grakn-dist`, can't we remove `grakn-bootup`? All the behaviour can instead go in `grakn-dist`. It makes sense to me that `grakn-engine` isn't responsible for the PID files.

Reply:
w.r.t PID file

That was what we used to do, however, PID file has to be managed by the process who owns it. Imagine a scenario where Grakn died due to out of memory long after the bootup script has completed.

Reply:
Got it! I can see why it has to be in `grakn-engine` now.